lib/Server.js

```diff
@@ -242,6 +242,15 @@ class Server {
           ? proxyConfigOrCallback()
           : proxyConfigOrCallback;
 
+      // Catch WebSocket errors by default
+      if (proxyConfig.ws && typeof proxyConfig.onProxyReqWs !== 'function') {
+        proxyConfig.onProxyReqWs = (proxyReq, req, socket) => {
+          socket.on('error', (err) => {
+            this.logger.warn('WebSocket error', err.message);
```

Comment:
I think we should add better message here, like `[http-proxy-middleware] Web socket error:` to avoid misleading

Reply:
My output looks like this:
```
<i> [webpack-dev-server] [HPM] Upgrading to WebSocket
<w> [webpack-dev-server] WebSocket error read ECONNRESET
<i> [webpack-dev-server] [HPM] Upgrading to WebSocket
<i> [webpack-dev-server] [HPM] Upgrading to WebSocket
<w> [webpack-dev-server] WebSocket error write after end
<w> [webpack-dev-server] WebSocket error write after end
<i> [webpack-dev-server] [HPM] Upgrading to WebSocket
<w> [webpack-dev-server] WebSocket error write after end
<w> [webpack-dev-server] WebSocket error write after end
```

Are you saying to change it to `this.logger.warn('WebSocket error:', err.message);` to make it:
```
<w> [webpack-dev-server] WebSocket error: write after end
```
or add `[http-proxy-middleware]` also to make it:
```
<w> [webpack-dev-server] [http-proxy-middleware] WebSocket error: write after end
```